Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /app
 
 # build typed-ast in separate stage because it requires gcc and libc-dev
 FROM base AS python-deps
-RUN apt-get update -qqy && apt-get install -qqy gcc libc-dev
+RUN apt-get update -qqy && apt-get install -qqy gcc libc-dev git
```

Comment:
Installing Git in the `python-deps` build stage won't make a difference in the final image.

I think it needs to be installed in the final output build, which has `apt-get install ...` on line 29.